### ISSUE TITLE
Test not failing if url isn't correct 

### ISSUE THREAD

**Issue by yeholerner** [kbbgl/e2ewd#16]
Added file
[log.log](https://github.com/kbbgl/e2ewd/files/3064205/log.log)

**Comment by kbbgl** [comment]
```
10/04/2019 10:01:13.063 [main] ERROR tests.MainTest - Failed to execute REST API tests: www1.cesm3dv.com
10/04/2019 10:01:13.064 [main] DEBUG tests.MainTest - [java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method), java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:929), java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1324), java.net.InetAddress.getAllByName0(InetAddress.java:1277), java.net.InetAddress.getAllByName(InetAddress.java:1193), java.net.InetAddress.getAllByName(InetAddress.java:1127), org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45), org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:111), org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353), org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380), org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236), org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184), org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88), org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110), org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184), org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82), org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:107), org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55), tests.SisenseRESTAPIClient.executeQuery(SisenseRESTAPIClient.java:63), tests.MainTest.executeRESTAPITests(MainTest.java:144), tests.MainTest.run(MainTest.java:95), tests.MainTest.init(MainTest.java:44), App.main(App.java:63)]
```

**Comment by kbbgl** [comment]
Fixed in version 2.4.95732.

Added termination for both REST API and MonetDB test throwing exceptions:

```
try {
                    executeMonetDBTest(elastiCube);

                } catch (InterruptedException e) {
                    logger.error("Error running MonetDB test: " +e.getMessage());
                    logger.debug(Arrays.toString(e.getStackTrace()));
                    testSuccess = false;
                    terminate("Error running MonetDB test: " + e.getMessage());
                }
```
And 
```
try {
                    executeRESTAPITests();

                } catch (IOException e) {
                    logger.error("Failed to execute REST API tests: " + e.getMessage());
                    logger.debug(Arrays.toString(e.getStackTrace()));
                    testSuccess = false;
                    terminate("Error running MonetDB test: " + e.getMessage());
                }
```
